src/components/theme/Anontools/Anontools.jsx

```diff
@@ -21,7 +21,7 @@ import { settings } from '~/config';
  * @class Anontools
  * @extends Component
  */
-export default class Anontools extends Component {
+class Anontools extends Component {
```

Comment:
Why is this required?

Reply:
This is for those decorators warning. I'll revert them, we can have a look at proptypes later